src/controllers/answer.js

```diff
@@ -0,0 +1,36 @@
+/* eslint no-unused-vars: ["error", { "argsIgnorePattern": "next" }] */
+import data from '../data.json';
+
+/** Class representing a question. */
+class Answers {
+  /**
+   * @param {request} req
```

Comment:
Nice work, documentation is very important and I applaud your effort in writing them. However, since you are required to use JSDoc for documentation, the words inside the curly braces should tell the data type of the params. You can visit this link for more info http://usejsdoc.org/about-getting-started.html